core/src/io/anuke/mindustry/world/blocks/distribution/ArmoredConveyor.java

```diff
@@ -17,8 +17,12 @@ public boolean acceptItem(Item item, Tile tile, Tile source){
 
     @Override
     public boolean blends(Tile tile, int rotation, int otherx, int othery, int otherrot, Block otherblock){
-        return otherblock.outputsItems() && (Point2.equals(tile.x + Geometry.d4(rotation).x, tile.y + Geometry.d4(rotation).y, otherx, othery)
-        || ((!otherblock.rotate && Edges.getFacingEdge(otherblock, otherx, othery, tile) != null &&
-            Edges.getFacingEdge(otherblock, otherx, othery, tile).relativeTo(tile) == rotation) || (otherblock.rotate && Point2.equals(otherx + Geometry.d4(otherrot).x, othery + Geometry.d4(otherrot).y, tile.x, tile.y))));
+        return otherblock.outputsItems() && blender(tile, rotation, otherx, othery, otherrot, otherblock);
+    }
+
+    public static boolean blender(Tile tile, int rotation, int otherx, int othery, int otherrot, Block otherblock){
```

Comment:
Since this is used by multiple `Autotile`-ing blocks, I would put this in the `Autotiler` trait as a default method and call it `armoredBlend` or something.